config/jobs/ppc64le-cloud/build-docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,166 @@
+periodics:
+    - name: periodic-config-docker
+      cluster: k8s-ppc64le-cluster
+      cron: 0 0 * * *
+      decorate: true
+      decoration_config:
+        timeout: 2h
```

Comment:
The default 2h timeout is picked up from here https://github.com/ppc64le-cloud/test-infra/blob/master/config/prow/config.yaml#L74.
Needn't be mentioned in this file.

Reply:
Thanks for that, I have removed it from the job.